source/install/desktop-app-changelog.rst

```diff
@@ -6,19 +6,70 @@ Desktop application changelog
 
 Latest Mattermost Desktop App releases:
 
-- `Release v5.4 <#id1>`_
-- `Release v5.3 <#id3>`_
-- `Release v5.2 <#id16>`_
-- `Release v5.1 <#id32>`_
-- `Release v5.0 <#id47>`_
-- `Release v4.7 <#id62>`_
+- `Release v5.5 <#id1>`_
+- `Release v5.4 <#id3>`_
+- `Release v5.3 <#id15>`_
+- `Release v5.2 <#id28>`_
+- `Release v5.1 <#id44>`_
+- `Release v5.0 <#id59>`_
+
+Release v5.5
+--------------
+
+**Release Day: September 15, 2023**
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+**Note:** Mattermost v5.5.0 contains a medium severity level security fix. Upgrading is highly recommended. Details will be posted on our `security updates page <https://mattermost.com/security-updates/>`__ 30 days after release as per the `Mattermost Responsible Disclosure Policy <https://mattermost.com/security-vulnerability-report//>`__.
+
+Compatibility
+~~~~~~~~~~~~~~~
+
+- Desktop App is supported on any supported Extended Support Release or a newer Mattermost server version.
```

Comment:
Curious: 
- Is the Desktop App really supported on _any_ ESR release? Or just the most recent ones? 
- Do we have a clear sense of what a "newer MM server version" indicates/suggests?

Reply:
I edited it to "... any currently supported Mattermost server version".